system-tests/projects/migration-e2e-coffeescript/expected-cypress.config.js

```diff
@@ -0,0 +1,11 @@
+const { defineConfig } = require('cypress')
```

Comment:
Nit: Should the `support/index.ts` above be `support.coffee`? Same with plugins. This is a coffeescript project, after all.

Reply:
Good catch, testing for this case showed that we needed to update the regex there as well